### ISSUE TITLE
I want to detect the flame，how to train from scratch?

### ISSUE THREAD

**Issue by Fatsheepgogogo** [fizyr/keras-retinanet#852]
I have tried transfer learning, but the results is not ideal. The AP is 0.56 and the loss is 0.1. My VOC datasets include 2k flame pictures and the resolution is not exactly the same.In fact，I have 10k flame pictures but the labeling work has not been completed yet. Running Debug.py but no annotations are displayed .When I transform VOC to CSV, debug.py shows the datasets are normal. Unfortunately training on CSV datasets I get loss value but no the AP value.
I test the inference model，when the label-to-name just is flame ,it can predict successfully.But if I add the class 'flame' in ‘label-to-names’ , the flame is highly likely to be predicted as person.
Because the features of flame are different from those of other objects ,can retinanet work effectively?
I would appreciate it if you could help me.

**Comment by luvWY** [comment]
Just change data processing... Make your dataset as VOC format and change the pascal_voc.py of the voc_classes dictionary.. Or you can simple use your own dataset as csv format.. Read me tell u everything

**Comment by hgaiser** [comment]
> Running Debug.py but no annotations are displayed

This is cause for concern, I'd check this before anything if I were you.

> When I transform VOC to CSV, debug.py shows the datasets are normal. Unfortunately training on CSV datasets I get loss value but no the AP value.

Did you use `--val-annotations` ?

> But if I add the class 'flame' in ‘label-to-names’ , the flame is highly likely to be predicted as person.

But your dataset consists only of flame? How can it be predicted as person?

> Just change data processing... Make your dataset as VOC format and change the pascal_voc.py of the voc_classes dictionary.

Ideally you wouldn't need to change the code, you can pass the classes to the constructor of the VOC generator to override the classes.

Closing this issue as it is not an issue with the code base.
